tests/integration/project/elements/sandbox-bwrap/build-dev-shm-not-mounted.bst

```diff
@@ -0,0 +1,11 @@
+kind: manual
+
+depends:
+- base.bst
+
+config:
+  create-dev-shm: false
+
+  build-commands:
+  - |
+    mountpoint -q /dev/shm || true
```

Comment:
This will always succeed. To ensure it's not mounted we need something like:
```! mountpoint -q /dev/shm```

Reply:
Done